pom.xml

```diff
@@ -2082,23 +2082,6 @@
           <artifactId>maven-deploy-plugin</artifactId>
           <version>2.8.2</version>
         </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-dependency-plugin</artifactId>
-          <executions>
-            <execution>
-              <id>default-cli</id>
-              <goals>
-                 <goal>build-classpath</goal>
-              </goals>
-              <configuration>
```

Comment:
This setting-change is needed in order for the dev/dependecies files to be generated properly, but I don't think we need the execution. Is there a way to keep the configuration change so that the `dev/test-dependencies` script continues to work but remove the default execution so that it's no longer automatically triggered?